ec2/multi_node/README.md

```diff
@@ -0,0 +1,4 @@
+# Multi-Node AWS OpenShift Deployment 
+OpenShift environment with the Service Catalog & Ansible Service Broker configured in AWS with multiple EC-2 instances
+
+TBD
```

Comment:
What is this TBD here for?

Reply:
I can remove it.. the contents will be updated once the files for the ec2 multi configuration gets placed in there